install

```diff
@@ -69,6 +69,7 @@ fzf_base="$(pwd)"
 ask() {
   while true; do
     read -p "$1 ([y]/n) " -r
+    REPLY=${REPLY:="y"}
```

Comment:
Although this works, the assignment is technically redundant. So we can instead write 1. `: ${REPLY:="y"}` which is more concise, 2. or `REPLY=${REPLY:-"y"}` to remove implicit assignment. What do you think?

Reply:
I usually do the second version.

Reply:
Me too, it clearly shows the intention.